pipeline/create_raw_pipeline.py

```diff
@@ -16,20 +17,31 @@
 from pipeline.transforms.compute_adjacency import ComputeAdjacency
 from pipeline.transforms.compute_encounters import ComputeEncounters
 from pipeline.transforms.resample import Resample
-from pipeline.transforms.write_date_sharded import WriteDateSharded
+from pipeline.transforms.writers import WriteEncountersToBQ
 from pipeline.transforms.readers import ReadSources
+from pipeline.utils.ver import get_pipe_ver
 
 import datetime
 import logging
 import pytz
 
 
-
 RESAMPLE_INCREMENT_MINUTES = 10.0
 MAX_GAP_HOURS = 1.0
 PRECURSOR_DAYS = 1
 
 
+def get_description(options: CreateOptions):
+    return f"""\
+    Created by the encounters_pipeline: {get_pipe_ver()}.
```

Comment:
just something super minor: The indent of the description here it'll be kept in the description of the BQ table, do we want that?